### PR TITLE
[Github actions]Fixed unit test

### PR DESCRIPTION
**Pull request by tuteng**
### Motivation

* Github actions fixed

### Modifications

* Add timeout configuration
* Use new mvn command for test 

### Verifying this change

Local test 
To do github action test

### CODE REVIEW

File: .github/workflows/ci-unit.yaml [hunk 1]
```diff
@@ -46,17 +46,34 @@ jobs:
         with:
           maven-version: 3.6.1
 
+      - name: run unit tests install by skip tests
+        run: mvn clean install -DskipTests
+
+      - name: run unit tests pulsar proxy
+        run: mvn install -DfailIfNoTests=false '-Dtest=!ProxyTest,!ProxyLookupThrottlingTest' -pl pulsar-proxy
```
Comment (aahmed-se) [inline .github/workflows/ci-unit.yaml]: no need to install not just run mvn test for all subsequent actions

File: .github/workflows/ci-unit-adaptors.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Maven
+        uses: aahmed-se/setup-maven@v3
```
Comment (rdhabalia) [inline .github/workflows/ci-unit-adaptors.yml]: we should avoid individual name mention here?

File: .github/workflows/ci-unit.yaml (hunk 1 above)
Comment (tuteng) [reply .github/workflows/ci-unit.yaml]: @aahmed-se PTAL

File: .github/workflows/ci-unit-adaptors.yml (hunk 2 above)
Comment (tuteng) [reply .github/workflows/ci-unit-adaptors.yml]: @rdhabalia I have created an issue https://github.com/apache/pulsar/issues/6012 for track remove this. PTAL

**Comment by tuteng** [comment]
There are too many unit tests for Github's actions to pass, and the tests and tests will affect each other. This pull request has already solved some of the problems. I think we can merge this pr and let GitHub actions run first, then we can solve all the problems step by step later.

**Comment by tuteng** [comment]
run java8 tests

**Comment by tuteng** [comment]
run integration tests

**Comment by jiazhai** [comment]
merge this first. will continually improve github action

**Pull request merged**